cypress/e2e/apitest/customers_request.cy.js

```diff
@@ -0,0 +1,100 @@
+/// <reference types="cypress" />
+
+
+
+describe('All test intermediary API',() => {
+    context('Request all customers', () => {
+        beforeEach('Get request', () => {
+            cy.request(Cypress.env('baseUrl')).as('customers')
+        })
```

Comment:
```suggestion
        })

```

Recomendo deixar uma linha em branco entre cada caso de teste (cada bloco `it`), para mais claramente separar os teste, melhorando a legibilidade.